packages/rn-tester-e2e/jest.setup.js

```diff
@@ -0,0 +1,31 @@
+const wdio = require('webdriverio');
+import capabilities from './e2e-config.js';
+import { beforeEach, afterEach, jest } from '@jest/globals';
+
+
+jest.setTimeout(40000);
```

Comment:
the timeout is already set to `60000` in `jest.config.js`, any reason why it's re-set here and it's a different value?

Reply:
yep, removed this one and keep the value in jest.config.js